puppet/modules/chassis/templates/local-config-db.php.erb

```diff
@@ -13,14 +13,19 @@ defined( 'ABSPATH' ) or define( 'ABSPATH', '<%= @wpdir %>/' );
 defined( 'WP_CONTENT_DIR' ) or define( 'WP_CONTENT_DIR', '<%= @contentdir %>' );
 
 <% if @network != false %>
-define( 'MULTISITE', true );
+if ( ! defined( 'WP_INSTALLING' ) || ! WP_INSTALLING ) {
+	define( 'MULTISITE', true );
```

Comment:
It's already inside `<% if @network != false %>` just above, which is the internal name for the `multisite` config option?